v63004/u_inref/gtm8874.csh

```diff
@@ -0,0 +1,95 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo '# Testing VIEW "[NO]STATSHARE"[:<region-list>]'
+
+echo '#### testA ####'
+echo '# enables or disables database statistics sharing for listed regions which permit such sharing. '
+echo '#'
+echo '# Without the region-list, the command acts on all regions enabled for sharing. '
+echo '#'
+echo '# When a targeted region has sharing disabled, STATSHARE has no immediate effect, '
+echo '# but may cause the process to start sharing when MUPIP SET -STATS enables sharing for that region'
+echo ''
+
+echo "# Create a 3 region DB with gbl_dir mumps.gld and regions DEFAULT, AREG, and BREG"
+$gtm_tst/com/dbcreate.csh mumps 3 >>& dbcreate_log_1.txt
+
+echo ''
+echo '# Disable sharing for BREG'
+$MUPIP set -NOSTAT  -reg "BREG" #>>& dbcreate_log.txt
+
+echo '# Run testA of gtm8874.m to test basic VIEW functionality and disabled regions'
+$ydb_dist/mumps -run testA^gtm8874
+
+echo '# Shut down the DB and backup necessary files to sub directory'
+$gtm_tst/com/dbcheck.csh >>& dbcreate_log_1.txt
+$gtm_tst/com/backup_dbjnl.csh dbbkup1 "*.gld *.mjl* *.mjf *.dat" cp nozip
+echo ''
+
+echo '#### testB ####'
+echo '# When a targeted region has sharing disabled, STATSHARE has no immediate effect, '
+echo '# but may cause the process to start sharing when MUPIP SET -STATS enables sharing for that region'
+echo '#'
+echo '# the $gtm_statshare environment variable applies to databases as the application first uses them'
+echo ''
+
+echo '# setenv gtm_statshare "TRUE"'
+setenv gtm_statshare "TRUE"
+echo '# Recreate the 3 region DB with gbl_dir mumps.gld'
+$gtm_tst/com/dbcreate.csh mumps 3 >>& dbcreate_log_2.txt
+
+echo ''
+echo '# Disable sharing for BREG'
+$MUPIP set -NOSTAT  -reg "BREG" #>>& dbcreate_log.txt
+
+echo '# Run testB1 of gtm8874.m to V[IEW] "STATSHARE" and run $VIEW for each region with gtm_statshare="TRUE"'
```

Comment:
Lines 57, 60 and 63 should be done by the same M program. That is, the MUPIP SET -STAT -reg BREG should be done while the M program has b.dat open and we need to verify that the $VIEW('STATSHARE","BREG") output changes from 0 to 1.

Reply:
@JakeYottaDB : As discussed, turns out that MUPIP SET -STATS requires standalone/exclusive access to the database so the below release note line is not relevant at all (one cannot change the STATS for a db from off to on while a process already has the db open). So please remove that from the test scripts.

*When a targeted region has sharing disabled, STATSHARE has no immediate effect, but may cause the process to start sharing when MUPIP SET -STATS enables sharing for that region.*

Reply:
The appropriate sections of the test have now been removed